.github/workflows/automated-zip-to-pr.yml

```diff
@@ -0,0 +1,124 @@
+name: Automated Zip to PR
+
+on:
+  issues:
+    types: [ "opened", "edited" ]
+
+  issue_comment:
+    type: [ "created", "edited" ]
+
+jobs:
+
+ extrakt:
+
+    # We don't do anything on pull requests, which are also considered issues
+    if: ${{ !github.event.issue.pull_request }}
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      # Only do these if the target is the ISSUE not a COMMENT on the issue
+      - name: Extract Zip URL (body)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ !github.event.comment.body }}
+        id: match-url
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '\((.*?\.zip)\)'
+      
+      - name: Extract Zip Filename (body)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ !github.event.comment.body }}
+        id: match-filename
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '\(.*\/(.*?\.zip)\)'
+
+      # We do this one ALWAYS so we can compare the issue zip name to the comment zip name
+      - name: Extract Theme Name (body)
+        uses: actions-ecosystem/action-regex-match@v2
+        id: match-theme-name
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '\(.*\/(.*?)\.zip\)'
```

Comment:
it will be nice to iterate later to bring THEME_NAME from `style.css` instead of zip name. (if there is no such name it can fallback to zip name)